### DISCUSSION TITLE
Representation model & other languages then English

### DISCUSSION THREAD

**Discussion by bfisseler** [MaartenGr/BERTopic#2231] [Q&A]
Hi, I want to use a local LLM as part of a representation model. The Examples all utilize models primarily used for English. But I have text in German. Can I use models like "zephyr-7b-alpha.Q4_K_M.gguf" with topic models on German texts? Or do I need to use a LLM specifically trained for German? And if so, which model should I use? 
I already did a search and found models such as "[TheBloke/leo-hessianai-7B-chat-GGUF](https://huggingface.co/TheBloke/leo-hessianai-7B-chat-GGUF)" or "[Nekochu/Llama-3.1-8B-German-ORPO](https://huggingface.co/Nekochu/Llama-3.1-8B-German-ORPO/tree/gguf)". But perhaps there are other LLMs more appropriate?

**Comment by MaartenGr** [comment]
I would generally advise looking into multilingual models, like the larger Llama models or Command R. If models are multilingual, then German is often one of those languages. 

Also, I would highly advise simply trying them out and experiment to see whether they fit your particular use case, data, and language.
